web/e2e-tests/message-basics.test.ts

```diff
@@ -310,8 +321,8 @@ async function expect_all_direct_messages(page: Page): Promise<void> {
 async function test_narrow_by_clicking_the_left_sidebar(page: Page): Promise<void> {
     console.log("Narrowing with left sidebar");
 
-    await page.click((await get_stream_li(page, "Verona")) + " a");
-    await expect_verona_stream(page);
+    await page.click((await get_stream_li(page, "Verona")) + " .stream-name");
+    await expect_verona_stream_top_topic(page);
```

Comment:
Instead of modifying what this test is checking here, can we go to the interleaved view and check the same thing. Then, add another test which just checks that we are going to the top topic in verona on clicking the stream name from left sidebar.

Reply:
Just want to clarify if I understood it right.
Are you suggesting that instead of altering `expect_verona_stream`, we navigate to the interleaved view and verify the same condition ( **what same condition are referring to** ) there? then,  adding a separate test to ensure that clicking the stream name from the left sidebar takes us to the top topic in Verona?

Reply:
Yeah, basically what the updated code here does.